scripts/vite/plugins/blsBrowserPlugin.ts

```diff
@@ -0,0 +1,32 @@
+import path from "node:path";
+import {UserConfig, ConfigEnv, Plugin} from "vite";
+const __dirname = new URL(".", import.meta.url).pathname;
+const polyfillsDir = path.join(__dirname, "../polyfills");
+
+const emptyModulePath = path.join(__dirname, "../polyfills/emptyModule.js");
+
+export function blsBrowserPlugin(): Plugin {
+  return {
+    name: "blsBrowserPlugin",
+    config(_config: UserConfig, _env: ConfigEnv) {
+      return {
+        optimizeDeps: {
+          exclude: ["@chainsafe/bls-keygen"],
```

Comment:
Curious what this does?

Reply:
It was supposed to exclude this package completely from the bundle, but didn't worked that way. Had to replace with the alias. Will remove this line.